Simulation.java

```diff
@@ -13,19 +13,19 @@
 	private double tmrca;
 	private double netau;
 	private double serialInterval;
-	private double antigenicDiversity;	
-	
+	private double antigenicDiversity;
```

Comment:
Can you reformat the entire repo once so we get all of these non-functional changes out of the way?